lib/routes/user/embedded_widgets/bitrefill_widget.dart

```diff
@@ -0,0 +1,83 @@
+import 'dart:convert' as JSON;
+import 'package:flutter/material.dart';
+import 'package:breez/bloc/blocs_provider.dart';
+import 'package:breez/bloc/invoice/invoice_bloc.dart';
+import 'package:flutter_webview_plugin/flutter_webview_plugin.dart';
+import 'package:breez/theme_data.dart' as theme;
+import 'package:breez/widgets/back_button.dart' as backBtn;
+
+class BitrefillPage extends StatefulWidget {
+  final String _title = "Bitrefill";
+
+  @override
+  State<StatefulWidget> createState() {
+    return new BitrefillPageState();
+  }
+}
+
+class BitrefillPageState extends State<BitrefillPage> {
+  final _widgetWebview = new FlutterWebviewPlugin();
+  String _lightningLink;
+
+  @override
+  void initState() {
+    super.initState();
+    _lightningLink = null;
+
+    _widgetWebview.lightningLinkStream.listen((value) {
```

Comment:
lightningLinkStream is not a good name, it is lightning specific inside a generic webview. Maybe a better name that reflects the purpose of this stream may be "postMessageStream" ?